### PR TITLE
follow up fix for Barbershop-Women template

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Needed for #5770 to really work

### CODE REVIEW

File: share/templates/convert.json [hunk 1]
```diff
@@ -10,9 +10,10 @@
 { "in": "02-Choral/05-SATB_Closed_Score_+_Organ.mscx",         "out": "02-Choral/05-SATB_Closed_Score_+_Organ.mscz"},
 { "in": "02-Choral/06-SATB_Closed_Score_+_Piano.mscx",         "out": "02-Choral/06-SATB_Closed_Score_+_Piano.mscz"},
 { "in": "02-Choral/07-Voice_+_Piano.mscx",                     "out": "02-Choral/07-Voice_+_Piano.mscz"},
-{ "in": "02-Choral/08-Barbershop_Quartet.mscx",                "out": "02-Choral/08-Barbershop_Quartet.mscz"},
-{ "in": "02-Choral/09-Liturgical_Unmetrical.mscx",             "out": "02-Choral/09-Liturgical_Unmetrical.mscz"},
-{ "in": "02-Choral/10-Liturgical_Unmetrical_+_Organ.mscx",     "out": "02-Choral/10-Liturgical_Unmetrical_+_Organ.mscz"},
+{ "in": "02-Choral/08-Barbershop_Quartet-(Men).mscx",          "out": "02-Choral/08-Barbershop_Quartet-(Men).mscz"},
+{ "in": "02-Choral/09-Barbershop_Quartet-(Women).mscx",        "out": "02-Choral/09-Barbershop_Quartet-(Women).mscz"},
```
Comment (Howard-C) [inline share/templates/convert.json]: I guess the hyphen before `(Men)` or `(Women)` should be underscore instead?

Comment (Jojo-Schmitz) [reply share/templates/convert.json]: darn... thanks for catchint this

Comment (Jojo-Schmitz) [reply share/templates/convert.json]: fixed

File: share/instruments/instrumentsxml.h [hunk 2]
```diff
@@ -11,7 +11,8 @@ QT_TRANSLATE_NOOP("Templates", "SATB Closed Score"),
 QT_TRANSLATE_NOOP("Templates", "SATB Closed Score + Organ"),
 QT_TRANSLATE_NOOP("Templates", "SATB Closed Score + Piano"),
 QT_TRANSLATE_NOOP("Templates", "Voice + Piano"),
-QT_TRANSLATE_NOOP("Templates", "Barbershop Quartet"),
+QT_TRANSLATE_NOOP("Templates", "Barbershop Quartet (Men)"),
+QT_TRANSLATE_NOOP("Templates", "Barbershop Quartet (Women)"),
```
Comment (Jojo-Schmitz) [inline share/instruments/instrumentsxml.h]: this is the vital part of this PR

File: share/templates/CMakeLists.txt [hunk 3]
```diff
@@ -17,6 +17,10 @@
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #=============================================================================
 
+# adding or renaming templates here should also better be done in convert.json, 
+# but definitly need to get reflected in ../instruments/instrumentsxml.h,
```
Comment (Howard-C) [inline share/templates/CMakeLists.txt]: Typo in `definitly` ;-)

File: share/templates/CMakeLists.txt [hunk 4]
```diff
@@ -17,6 +17,10 @@
...
+# like by running ../instruments/generateTs.py
+
```
Comment (Jojo-Schmitz) [inline share/templates/CMakeLists.txt]: this part is to prevent issues like this from happening again

File: share/templates/convert.json [hunk 5]
```diff
@@ -10,9 +10,10 @@
 { "in": "02-Choral/05-SATB_Closed_Score_+_Organ.mscx",         "out": "02-Choral/05-SATB_Closed_Score_+_Organ.mscz"},
 { "in": "02-Choral/06-SATB_Closed_Score_+_Piano.mscx",         "out": "02-Choral/06-SATB_Closed_Score_+_Piano.mscz"},
 { "in": "02-Choral/07-Voice_+_Piano.mscx",                     "out": "02-Choral/07-Voice_+_Piano.mscz"},
-{ "in": "02-Choral/08-Barbershop_Quartet.mscx",                "out": "02-Choral/08-Barbershop_Quartet.mscz"},
-{ "in": "02-Choral/09-Liturgical_Unmetrical.mscx",             "out": "02-Choral/09-Liturgical_Unmetrical.mscz"},
-{ "in": "02-Choral/10-Liturgical_Unmetrical_+_Organ.mscx",     "out": "02-Choral/10-Liturgical_Unmetrical_+_Organ.mscz"},
+{ "in": "02-Choral/08-Barbershop_Quartet_(Men).mscx",          "out": "02-Choral/08-Barbershop_Quartet_(Men).mscz"},
+{ "in": "02-Choral/09-Barbershop_Quartet_(Women).mscx",        "out": "02-Choral/09-Barbershop_Quartet_(Women).mscz"},
+{ "in": "02-Choral/10-Liturgical_Unmetrical.mscx",             "out": "02-Choral/10-Liturgical_Unmetrical.mscz"},
+{ "in": "02-Choral/11-Liturgical_Unmetrical_+_Organ.mscx",     "out": "02-Choral/11-Liturgical_Unmetrical_+_Organ.mscz"},
```
Comment (Jojo-Schmitz) [inline share/templates/convert.json]: and this part is mostly optional, as convert.json isn't used currently

File: share/templates/CMakeLists.txt (hunk 3 above)
Comment (Jojo-Schmitz) [reply share/templates/CMakeLists.txt]: fixed

Comment (Jojo-Schmitz) [reply share/templates/CMakeLists.txt]: How many typos can a single person to in a single PR of this small size ;-(

File: share/templates/CMakeLists.txt [hunk 6]
```diff
@@ -17,6 +17,10 @@
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #=============================================================================
 
+# adding or renaming templates here should also better be done in convert.json, 
+# but definitely need to get reflected in ../instruments/instrumentsxml.h,
```
Comment (Howard-C) [inline share/templates/CMakeLists.txt]: Sorry, but I think the correct grammar should be: `but definitely needs to`? :-)

Comment (Jojo-Schmitz) [reply share/templates/CMakeLists.txt]: `adding or renamiming templates`, plural here, so IMHO correct there

Comment (Howard-C) [reply share/templates/CMakeLists.txt]: `or`, so that's a single event chosen between two.

**Pull request merged**